1.9/administration/upgrading.md

```diff
@@ -19,7 +19,7 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 
 - Mesos, Mesos Frameworks, Marathon, Docker and all running tasks in the cluster should be stable and in a known healthy state.
 - For Mesos compatibility reasons, we recommend upgrading any running Marathon-on-Marathon instances to version Marathon 0.16.0-RC3 before proceeding with this DC/OS upgrade.
```

Comment:
That hasn't been updated for a while I guess. The recommended Marathon Version on DC/OS 1.8 is 1.3.5.